src/main/java/robosky/uplands/block/unbreakable/UnbreakableBlock.java

```diff
@@ -0,0 +1,24 @@
+package robosky.uplands.block.unbreakable;
+
+import net.minecraft.block.Block;
+
+/**
+ * A plain unbreakable block.
+ */
+public class UnbreakableBlock extends Block implements Unbreakable {
+
+    private final Block base;
+
+    public UnbreakableBlock(Block base) {
+        super(Block.Settings.copy(base).strength(-1.0f, 3600000.0f));
```

Comment:
Hmm, doesn't need to be done now, but could it be possible to forward this block's calls into the base type?

Reply:
It's possible but I question whether this would be useful for only a basic unbreakable block. Maybe we can add `getOutlineShape` and such, but it's not necessary now.

Reply:
yea the outline shape makes sense. I'd say add em as they are needed then.